4/main.js

```diff
@@ -0,0 +1,123 @@
+const url = 'https://opentdb.com/api.php?amount=10&type=multiple';
+const title = document.getElementById ('title');
+const quizLevel = document.getElementById ('quizLevel');
+const text = document.getElementById ('text');
+const startBtn = document.getElementById ('btn');
+const btnArea = document.getElementById ('btnArea');
+let quizList;
+let curIndex = 0;
+let correctAnswers = 0;
+
+btn.addEventListener ('click', getQuizList);
+
+// クイズデータの取得と1問目のクイズのセット
+async function getQuizList () { 
+  title.textContent = '取得中';
+  text.textContent = '少々お待ちください';
+  btnArea.removeChild (startBtn);
+  
+  const json = await fetch (url)
+  .then (response => {
+    return response.json ();
+  }).catch (error => {
+    console.log ('取得失敗 ' + error); 
+    return null;
+  })
+
+  quizList = json.results;
+  quizSet ();
+}
+
+//クイズをセットする関数
+function quizSet () {
```

Comment:
メソッド、関数名は「自動詞」または「他動詞 + 目的語」とすることが多いです。
（プログラム言語の大半は英語圏で作られているので…）


```suggestion
function setQuiz () {
```